data/dfndata/items/misc/consumables.dfn

```diff
@@ -311,4 +311,24 @@ pileable=1
 amount=1
 decay=1
 origin=lbr
+}
+
+[base_recipe_scroll]
+{ 
+get=base_item
+name=recipe scroll
+id=0x2831
+weight=100
+pileable=1
+amount=1
+decay=1
+script=5061//recipe scroll
+origin=ml
+}
+
+[skullcap_recipe]
```

Comment:
Update the example recipe to grant user ability to craft an item they don't otherwise have access  to (skullcap is already available by default), so they can see the result of learning the example recipe.